.github/workflows/build-test.yml

```diff
@@ -52,6 +52,14 @@ jobs:
           name: Test Results - Unit
           path: test-result-*.xml
 
+      - name: Coverage
```

Comment:
Eventually we should get rid of this, as the tests are now run twice, aren't they? (Above in Unit test task)

If they are reported correctly, we can just use those results (?)

Reply:
Yes, true this is just a mid-step, next step would be a cleanup of the old stuff.